src/components/general/button/index.tsx

```diff
@@ -4,17 +4,27 @@ import { IconProp } from '@fortawesome/fontawesome-svg-core';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { ContextualColors } from '../../../styles.d';
 
-const StyledButton = styled.button<{ color: keyof ContextualColors }>`
+const StyledButton = styled.button<{
+  color: keyof ContextualColors;
+}>`
   width: 100%;
   height: fit-content;
-  background: ${({ theme, color }) => theme.palette[color].main};
+  background: ${({ theme, color }) => theme.paletteV2[color].background};
   color: ${({ theme, color }) => theme.palette[color].contrast};
+  box-shadow: 3px 3px ${({ color, theme }) => theme.paletteV2[color].shadow};
+  border: 1px solid ${({ color, theme }) => theme.paletteV2[color].borderColor};
+  border-radius: ${({ color, theme }) => theme.paletteV2[color].borderRadius};
   padding: ${({ theme }) => theme.sizes.spacing.md};
   font-size: ${({ theme }) => theme.sizes.fonts.md};
   outline: none;
-  border: 1px solid ${({ theme, color }) => theme.palette[color].contrast};
   cursor: pointer;
 
+  :hover {
```

Comment:
Looking good! ✨  Just thinking about the feedback from the testers that navigating the site with the keyboard was difficult, if you add a :focus here, will the styling also apply when you tab through the site?

Reply:
yeah just tried it works when you tab through and also hover, Nice Choice

Reply:
thing is once you click it it keeps the focus on the button and doesnt remove the hover styling. so you have to either click somewhere else on the screen to change focus

Reply:
oooh! does focus-visible work instead?

Reply:
nope still remains the focus on the button if tabbing through

Reply:
does it remove the styling when you click and then mouse leave?

hmm i'll have a google if there's a solution, it might be the behaviour for tabbing through, you don't want to click a button and lose your place and have to tab through the site again?